vcap/vcap/loading/capsule_loading.py

```diff
@@ -288,3 +293,36 @@ def capsule_module_name(data: bytes) -> str:
     """Creates a unique module name for the given capsule bytes"""
     hash_ = hashlib.sha256(data).hexdigest()
     return f"capsule_{hash_}"
+
+
+_TYPE_CALLABLE = Union[type, callable]
+
+
+def _validate_capsule_field(capsule: BaseCapsule,
+                            name: str,
+                            value: Any,
+                            type_: _TYPE_CALLABLE) -> None:
+    if not _check_type(value, type_):
+        raise InvalidCapsuleError(
+            f"The capsule has an invalid internal configuration!\n"
+            f"Capsule field {name} must be of type {type_}, got {type(value)}",
```

Comment:
This will print `Capsule field field_name must be of type <built-in function callable>, got...` if `type_` is `callable`

That might read a bit strange, as we're not expecting a built-in function.

Reply:
That's a good point.